mne/channels/montage.py

```diff
@@ -621,6 +621,7 @@ def _set_montage(info, montage, match_case=True):
         The measurement info to update.
     %(montage)s
     %(match_case)s
+    %(on_missing)s
```

Comment:
There is already an `on_missing` in `mne/utils/docs.py` and it is not the correct one for this use case. You should create a new `on_missing_montage` or so in `mne/utils/docs.py` and use it here and in `meas_info.py`